src/Sarif/CodeGenHints.json

```diff
@@ -360,11 +520,6 @@
       "kind": "DictionaryHint"
     }
   ],
-  "Result.Fingerprints": [
```

Comment:
>Result.Fingerprints [](start = 3, length = 19)

FYI, i realphabetized some content in this file, as it had gotten out of date based on some renaming. This construct has moved earlier in the file, for example.